### ISSUE TITLE
No safearea on IOS

### ISSUE THREAD

**Issue by Antoinewaag** [mmazzarolo/react-native-modal-datetime-picker#706]
## Environment

```
System:
    OS: macOS 13.2.1
    CPU: (8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz
    Memory: 69.79 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.18.1 - ~/.nvm/versions/node/v16.18.1/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 9.6.2 - ~/.nvm/versions/node/v16.18.1/bin/npm
    Watchman: 2023.04.17.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 2022.2 AI-222.4459.24.2221.9862592
    Xcode: 14.2/14C18 - /usr/bin/xcodebuild
  Languages:
    Java: javac 19 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0 
    react-native: 0.71.7 => 0.71.7 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

```

## Platforms

IOS

## Versions

- Android:
- iOS: 
- react-native-modal-datetime-picker: 15.0.1
- [react-native-community/datetimepicker](https://github.com/react-native-community/datetimepicker): 7.1.0
- react-native: 0.71.7
- react: 18.2.0

## Description

On IOS, there is no safearea (padding), so the button is stuck at the bottom of the screen.

Do I have to add it manually with `modalStyleIOS` and `padding` ? Is it not a feature which should be included in the package ?

## Screenshot

<img width="409" alt="Capture d’écran 2023-05-30 à 08 47 36" src="https://github.com/mmazzarolo/react-native-modal-datetime-picker/assets/51699943/a7f73e83-86b9-47fb-98bf-c603ff972641">

**Comment by kolking** [comment]
@mmazzarolo I've made https://github.com/mmazzarolo/react-native-modal-datetime-picker/pull/718 to resolve the issue.

**Comment by mmazzarolo** [comment]
Fixed by #718
